components/expression_language/syntax.rst

```diff
@@ -127,9 +127,9 @@ Working with Functions
 ----------------------
 
 You can also use registered functions in the expression by using the same
-syntax as PHP and JavaScript. The ExpressionLanguage component comes with one
-function by default: ``constant()``, which will return the value of the PHP
-constant::
+syntax as PHP and JavaScript. The ExpressionLanguage component comes with two
```

Comment:
Let's remove the "two" and use "following functions"...
This way we don't need to adapt it for further methods in the future